torch/export/exported_program.py

```diff
@@ -345,6 +346,7 @@ def _decompose_and_get_gm_with_new_signature_constants(
 
         fake_args_unwrapped = pytree.tree_unflatten(fake_args, mod._in_spec)
         fake_mode = detect_fake_mode(fake_args)
+        fake_mode = contextlib.nullcontext() if fake_mode is None else fake_mode
```

Comment:
Doesn't this make the code the same as the else case? where ep.verifier.dialect != "TRAINING"?

Reply:
The only difference is how we first find the fake_args. I plan to merge these two logic together somehow. Just testing CI first.

Reply:
+1